apps/ejabberd/cover.spec

```diff
@@ -1,4 +1,4 @@
-{export, "/tmp/mognoose_ct.coverdata"}.
+{export, "mognoose_ct.coverdata"}.
```

Comment:
This will break current coverage reporting. See [run_common_test.erl#L221](https://github.com/esl/MongooseIM/blob/db630b7385aac8376d7c8b538f6ee632a585789e/test/ejabberd_tests/run_common_test.erl#L221)